lametro/views.py

```diff
@@ -81,10 +81,12 @@ def get_context_data(self, **kwargs):
             context['packet_url'] = None
 
         # Create list of related board reports, ordered by descending last_action_date.
+        # Thanks https://stackoverflow.com/a/2179053 for how to handle null last_action_date
         if context['legislation'].related_bills.all():
             all_related_bills = context['legislation'].related_bills.all().values('related_bill_identifier')
             related_bills = LAMetroBill.objects.filter(identifier__in=all_related_bills)
-            context['related_bills'] = sorted(related_bills, key=lambda bill: bill.last_action_date, reverse=True)
+            minimum_date = datetime.date(datetime.MINYEAR, 1, 1)
+            context['related_bills'] = sorted(related_bills, key=lambda bill: bill.last_action_date or minimum_date, reverse=True)
```

Comment:
`can't compare offset-naive and offset-aware datetimes` happens (as you might expect) when you try to compare datetime objects with a timezone and without one. the datetimes in the metro db all have the los angeles timezone associated. so, you need to [add a timezone](https://docs.python.org/3/library/datetime.html#datetime-objects) to your minimum date.

n.b., `app_timezone` is [defined in models](https://github.com/datamade/la-metro-councilmatic/blob/730abffbc065c17d43a372b671d8c3c2bdce5b13/lametro/models.py#L19) and [imported in this module](https://github.com/datamade/la-metro-councilmatic/blob/730abffbc065c17d43a372b671d8c3c2bdce5b13/lametro/views.py#L36) with the `*`, so it's available for you!